LayoutTests/imported/w3c/web-platform-tests/fetch/api/headers/header-values-normalize.html

```diff
@@ -67,6 +67,6 @@
         assert_equals(res.headers.get("x-request-val3"), expectedVal3)
       })
     }
-  }, "fetch() with value " + encodeURI(val))
+  }, "fetch() with value 111 " + encodeURI(val))
```

Comment:
Committed a WPT change to help with testing. This will not be included in the bugzilla patch.